vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -63,8 +62,10 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
       batchingUIMessageQueue);
 
   modules.emplace_back(
-      react::uwp::WebSocketModule::Name,
-      []() { return std::make_unique<react::uwp::WebSocketModule>(); },
+      "WebSocketModule",
+      [context]() mutable {
```

Comment:
nit: if we're not mutating the capture, so this shouldn't need to be marked mutable. If this is marked mutable, we should be able to do a `std::move` of the context to avoid an extra atomic addref/release.

Reply:
Stale from a previous usage of `std::move`. Will remove.